CuraEngineJS/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@ultimaker/curaenginejs",
+  "version": "0.2.0",
+  "description": "CuraEngineJS a TS component to run CuraEngine in a browser",
+  "main": "src/CuraEngine.js",
+  "scripts": {
+    "install_curaengine": "conan install ${npm_package_config_conan_package} -s build_type=Release --build=missing --update -c tools.build:skip_test=True -pr:h cura_wasm.jinja -if src && rm -f src/*conan*",
```

Comment:
Do the install script now in the package.json but in the workflow and use the conan version id :arrow_up: